rmw_fastrtps_shared_cpp/src/rmw_take.cpp

```diff
@@ -86,6 +88,55 @@ _take(
   return RMW_RET_OK;
 }
 
+rmw_ret_t
```

Comment:
`static` ?

Reply:
Why is that?

Reply:
It looks like the definition of this function should only be used within this translation unit. But I also just realized this is C++ and we have a namespace too, so it's not a big deal.

Reply:
I think also the `_take` and `_assign_message_info` functions could also be `static`. But it's a bit outside the scope of this PR.